### DISCUSSION TITLE
There is no Vercel account linked with this GitHub account.

### DISCUSSION THREAD

**Discussion by behawesometw** [vercel/community#1745] [Help] [answered]
### Summary

My account suddenly cannot log in, and the same situation occurs whether it's through the website or cli. Has anyone else experienced this?

## Website login fail snapshot
<img width="422" alt="image" src="https://user-images.githubusercontent.com/60811700/224292461-0cbd7db2-60d4-4eae-9902-fd172b1dff2a.png">

## CLI login fail snapshot
<img width="660" alt="截圖 2023-03-10 下午6 27 51" src="https://user-images.githubusercontent.com/60811700/224292638-649bf91d-438e-4897-bb99-de25db8bca7a.png">

## Github repo permissions
<img width="1187" alt="image" src="https://user-images.githubusercontent.com/60811700/224293563-a64a32ae-10cd-4f6f-8d78-4c9bd175960a.png">

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @behawesometw. Please send an email to [registration@vercel.com](mailto:registration@vercel.com) and include the email address and GitHub username that should be linked to your account. That way the team can help you while keeping your personal information private 🙏
